site/pages/docs/ref/plugins-fr.hbs

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
Might be worth checking in Notepad++ to check that the encoding is still UTF-8 (without BOM)